src/test/scala/nl.knaw.dans.easy.deposit/SubmitterSpec.scala

```diff
@@ -41,7 +42,7 @@ class SubmitterSpec extends TestSupportFixture with MockFactory {
     .getOrElse(fail("could not get DOI from test input"))
 
   "constructor" should "fail if the configured group does not exist" in {
-    val props = minimalAppConfig.properties
+    val props = minimalAppConfig.properties.clone().asInstanceOf[PropertiesConfiguration]
```

Comment:
why is this? same in other (new) tests

Reply:
It's a mutable object and I don't want to influence other tests.

Reply:
ah, minimalAppConfig is a def, not a val